tequilapi/client/client.go

```diff
@@ -253,8 +253,8 @@ func (client *Client) GetSessions() (SessionsDTO, error) {
 	return sessions, err
 }
 
-// Services returns all running services.
```

Comment:
Just a notice, I'm using these dots at the end of sentence/comment by the recommendation of [Effective GO](https://golang.org/doc/effective_go.html#commentary).
For example, standard packages are using it everywhere: https://golang.org/src/errors/errors.go

Reply:
ok, when we need to make it consistent everywhere